src/main/java/seedu/academyassist/logic/commands/AddClassCommand.java

```diff
@@ -19,31 +20,28 @@ public class AddClassCommand extends Command {
 
     public static final String COMMAND_WORD = "addc";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a class to the student with this ic number. "
-            + "Parameters: \n- "
-            + PREFIX_IC + "IC \n- "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a class to the student with this student id. "
+            + "Parameters: Student ID (5-digit number)\n- "
             + PREFIX_SUBJECT + "Subject \n- "
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_IC + "S1234567A "
+            + PREFIX_IC + "12345 "
```

Comment:
Maybe here is supposed be a different prefix for studentID? There is a bit of confusion here.

Reply:
Arhh there shouldn't be a prefix here, will correct it.